src/dom.js

```diff
@@ -16,6 +16,60 @@
 
 
 /**
+ * Waits until the child element is constructed. Once the child is found, the
+ * callback is executed.
+ * @param {!Element} parent
+ * @param {function():boolean} checkFunc
+ * @param {function()} callback
+ */
+export function waitForChild(parent, checkFunc, callback) {
+  if (checkFunc()) {
+    callback();
+    return;
+  }
+  const win = parent.ownerDocument.defaultView;
+  if (win.MutationObserver) {
+    const observer = new win.MutationObserver(() => {
+      if (checkFunc()) {
+        observer.disconnect();
+        callback();
+      }
+    });
+    observer.observe(parent, {childList: true});
+  } else {
+    const interval = win.setInterval(() => {
+      if (checkFunc()) {
+        win.clearInterval(interval);
+        callback();
+      }
+    });
```

Comment:
Something like 5 is better I believe. I think that is what we typically use for "almost busy wait"